src/System.IO.FileSystem/tests/Directory/GetFileSystemEntries_str_str.cs

```diff
@@ -652,19 +784,32 @@ public void SearchPatternLongPath()
 
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]  // Search pattern with double dots throws ArgumentException
-        public void WindowsSearchPatternWithDoubleDots()
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+        public void WindowsSearchPatternWithDoubleDots_Desktop()
         {
             Assert.Throws<ArgumentException>(() => GetEntries(TestDirectory, Path.Combine("..ab ab.. .. abc..d", "abc..")));
             Assert.Throws<ArgumentException>(() => GetEntries(TestDirectory, ".."));
             Assert.Throws<ArgumentException>(() => GetEntries(TestDirectory, @".." + Path.DirectorySeparatorChar));
         }
 
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]  // Search pattern with double dots throws ArgumentException
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void WindowsSearchPatternWithDoubleDots_Core()
+        {
+            string directory = Directory.CreateDirectory(GetTestFilePath()).FullName;
+            Assert.Throws<DirectoryNotFoundException>(() => GetEntries(directory, Path.Combine("..ab ab.. .. abc..d", "abc..")));
+            GetEntries(directory, "..");
+            GetEntries(directory, @".." + Path.DirectorySeparatorChar);
+        }
+
         private static char[] OldWildcards = new char[] { '*', '?' };
         private static char[] NewWildcards = new char[] { '<', '>', '\"' };
 
         [Fact]
         [PlatformSpecific(TestPlatforms.Windows)]  // Windows-invalid search patterns throw
-        public void WindowsSearchPatternInvalid()
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
```

Comment:
Is it that we've made something which didn't use to work and threw an exception now instead work?  So we still throw on netfx and .NET Core on Unix but not on Windows?  Will this be made to work on Unix as well for consistency, and then presumably ported to netfx for consistency there as well?

Reply:
Yes, I missed removing this from Unix. We'll port to NetFX in full trust- this was a limited trust based block.